### PR TITLE
Import Travis CI docs from devs manual

### PR DESCRIPTION
**Pull request by DavidePrincipi**
https://github.com/NethServer/dev/issues/6230

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,25 +21,32 @@ On Fedora 31+ and CentOS 8, run as a non-root user ::
...
 
-Requisites to build RPMs starting from a git repository:
+- The .spec file is in the current working directory
 
-- The .spec file is placed at the root of the repository
+- Source tarballs defined in the .spec file can be downloaded via ``http://`` or ``ftp://``.
 
-- In the specfile, ``source0`` corresponds to the git archive output in
-  ``tar.gz`` format (e.g. ``Source: %{name}-%{version}.tar.gz``)
+- If a ``SHA1SUM`` file in the current working directory exists, the integrity of
```
Comment (gsanchietti) [inline README.rst:35]:
```suggestion
- If a ``SHA1SUM`` file exists in the current working directory, the integrity of
```

File: README.rst [hunk 2]
```diff
@@ -94,19 +101,267 @@ https://hub.docker.com/r/nethserver/makerpms.
...
+Configuration
+^^^^^^^^^^^^^
+
+To automate the RPM build process using Travis CI
+
+* create a ``.travis.yml`` file inside the source code repository hosted on
+  GitHub.
+
+* the repository must have Travis CI builds enabled and upload secrets properly set up.
+  Contact the organization maintainer on community.nethserver.org for help.
```
Comment (gsanchietti) [inline README.rst:233]:
```suggestion
  Contact the organization maintainer on `community.nethserver.org <https://community.nethserver.org>` for help.
```

File: README.rst [hunk 3]
```diff
@@ -94,19 +101,267 @@ https://hub.docker.com/r/nethserver/makerpms.
...
+- ``DOCKER_IMAGE``
+- ``DEST_ID``
+
+NSVER
+~~~~~
+
+``NSVER`` selects the target NethServer version for the build system. Currently
+the supported version values are ``7`` and ``6``.
+
+DOCKER_IMAGE
```
Comment (gsanchietti) [inline README.rst:317]: The `ŧravis.yml` of this repo uses the `IMAGE_REPO` variable not `DOCKER_IMAGE`.
Which is the right one?

File: README.rst [hunk 4]
```diff
@@ -94,19 +101,267 @@ https://hub.docker.com/r/nethserver/makerpms.
...
+
+- ``buildsys7`` is based on the previous environment. It also pulls in the dependencies for arch-dependant packages (like ``asterisk13`` or ``ns-samba``).
+  It actually installs the ``buildsys-build`` package group, which provides the ``gcc`` compiler (version 4) among other packages.
+
+- ``devtoolset7``: it extends the *buildsys7* with the devtoolset-9 SCLo packages set. It is possible to
+  compile with gcc version 9, by prefixing the container entry point in the following way: ::
+
+    docker run -ti [OPTIONS] scl enable devtoolset-9 -- makerpms-travis package.spec
+
+  See for instance https://github.com/NethServer/sofia-sip
```
Comment (gsanchietti) [inline README.rst:333]:
```suggestion
  See for instance ` sofia-sip pacakge <https://github.com/NethServer/sofia-sip>`_.
```

File: README.rst [hunk 5]
```diff
@@ -94,19 +101,267 @@ https://hub.docker.com/r/nethserver/makerpms.
...
+* Pull requests are uploaded to ``nethforge-autobuild``
+
+* Branch builds are uploaded to ``nethforge-testing``, whilst tagged builds are uploaded to ``nethforge``
+
+.. warning::
+
+    In any case, **the git tag must begin with a digit
+    and not containing any "-" minus symbol**.
+    For instance the tag ``0.1.12`` is considered
+    as a tagged build whilst ``v0.1.12`` not
```
Comment (gsanchietti) [inline README.rst:357]:
```suggestion
    as a tagged build whilst ``v0.1.12`` is not
```

File: README.rst (hunk 3 above)
Comment (DavidePrincipi) [reply README.rst:317]: They are two distinct things: 

- IMAGE_REPO is just a prefix used to build and push the container image to Docker hub.
- DOCKER_IMAGE is self-contained in the .travis.yml example for RPM builds.

Comment (gsanchietti) [reply README.rst:317]: :man_shrugging:

**Pull request merged**